src/java.base/share/classes/java/io/OutputStream.java

```diff
@@ -148,6 +145,12 @@ public void write(byte[] b) throws IOException {
      * {@code off+len} is greater than the length of the array
      * {@code b}, then an {@code IndexOutOfBoundsException} is thrown.
      *
+     * @implSpec
+     * The {@code write} method of {@code OutputStream} calls
+     * the write method of one argument on each of the bytes to be
+     * written out. Subclasses are encouraged to override this method and
```

Comment:
Shouldn't the "subclasses" information be part of the API Note?

Reply:
Sure; moved to an apiNote.